pkgs/servers/mastodon/update.sh

```diff
@@ -94,7 +94,7 @@ in
       rev = "v\${version}";
       hash = "$HASH";
     };
-    patches = [$PATCHES];
+    inherit patches;
```

Comment:
```suggestion
    patches = patches ++ [ $PATCHES];
```

The update script is capable of taking patches as arguments. To me this functionality is overkill, but let's not break that for now.

Reply:
Thanks for catching that! I just did a blind copy-paste in the script without ever wondering why there was an empty list in there.